app/views/admin/comments/index.html.erb

```diff
@@ -49,8 +49,8 @@
 <script>
   window.addEventListener('load', function() {
     $("form").submit(function (e) {
-      $(this).toggleClass("d-none");
-      $(this).toggleClass("d-inline");
+      $(this).toggleClass("hidden");
```

Comment:
In `Comments` section only these two [bootstrap classes](https://getbootstrap.com/docs/4.0/utilities/display/#examples) where found and I have updated them as per crayons.